simulation_ws/.rosinstall

```diff
@@ -1,2 +1,3 @@
 # kinetic.rosinstall
 - setup-file: {local-name: /opt/ros/kinetic/setup.sh}
+- git: {local-name: src/turtlebot_simulation, uri: "https://github.com/ROBOTIS-GIT/turtlebot3_simulations", version: 1.2.0}
```

Comment:
Keep the folder name consistent with the repository for simplicity. I.e, "src/turtlebot3_simulations"